tests/runner/origin/base_host.py

```
@@ -329,176 +342,159 @@ class RunHostAndProgramRes:
+					'max_execution_minutes': max_exec_mins,  # this parameter is needed to prevent zombie genvms
+					'timestamp': timestamp,
+					'host': host,
+					'extra_args': extra_args,
+				},
+			) as resp:
+				logger.debug('post /genvm/run', status=resp.status)
+				data = await resp.json()
+				logger.trace('post /genvm/run', body=data)
+				if resp.status != 200:
+					logger.error(
+						f'genvm manager /genvm/run failed', status=resp.status, body=data
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)